src/hints/math/testLessThanOrEqual.ts

```diff
@@ -0,0 +1,55 @@
+import { z } from 'zod';
+
+import { Felt } from 'primitives/felt';
+import { VirtualMachine } from 'vm/virtualMachine';
+
+import { HintName } from 'hints/hintName';
+import {
+  cellRef,
+  resOperand,
+  CellRef,
+  ResOperand,
+} from 'hints/hintParamsSchema';
+
+/** Zod object to parse TestLessThanOrEqual hint */
+export const testLessThanOrEqualParser = z
+  .object({
+    TestLessThanOrEqual: z.object({
+      lhs: resOperand,
+      rhs: resOperand,
+      dst: cellRef,
+    }),
+  })
+  .transform(({ TestLessThanOrEqual: { lhs, rhs, dst } }) => ({
+    type: HintName.TestLessThanOrEqual,
+    lhs,
+    rhs,
+    dst,
+  }));
+
+/**
+ * TestLessThanOrEqual hint
+ *
+ * Store true at `dst` if value at `lhs` is inferior or equal to value at `rhs`.
+ * Store false otherwise
+ */
+export type TestLessThanOrEqual = z.infer<typeof testLessThanOrEqualParser>;
+
+/**
+ * TestLessThanOrEqual hint
+ *
+ * Check whether the value at `lhs` is less than or equal to the value at `rhs`
+ *
+ * Store the boolean result (0 or 1) at `dst`
+ */
+export const testLessThanOrEqual = (
```

Comment:
use a proper JSDoc as in [`AllocateFelt252Dict`](https://github.com/kkrt-labs/cairo-vm-ts/blob/e133561a28ea1f269f6efd1af8bb657007c20b74/src/hints/dict/allocFelt252Dict.ts#L28-L42)